apps/backend/src/main.ts

```diff
@@ -1,9 +1,32 @@
 import { NestFactory } from '@nestjs/core';
+import { DocumentBuilder, SwaggerModule } from '@nestjs/swagger';
 import { AppModule } from './app.module';
+import { swaggerDocumentOptions } from './swagger/swagger.config';
 
 async function bootstrap() {
   const app = await NestFactory.create(AppModule);
 
+  const swaggerSettings = new DocumentBuilder()
+    .setTitle('Swagger Animavita')
+    .setDescription(
+      'This is a sample server Animavita server. You can find out more about Swagger at [http://swagger.io](http://swagger.io).',
+    )
+    .setVersion('1.0.0')
```

Comment:
I think here you're supposed to use `SWAGGER_VERSION` constant

Reply:
Done!